pkg/api/handlers/compat/containers_stop.go

```diff
@@ -23,9 +23,10 @@ func StopContainer(w http.ResponseWriter, r *http.Request) {
 
 	// /{version}/containers/(name)/stop
 	query := struct {
-		Ignore        bool `schema:"ignore"`
-		DockerTimeout uint `schema:"t"`
-		LibpodTimeout uint `schema:"timeout"`
+		Ignore        bool                `schema:"ignore"`
+		DockerTimeout uint                `schema:"t"`
+		LibpodTimeout uint                `schema:"timeout"`
+		Filter        map[string][]string `schema:"filters"`
```

Comment:
The new `filters` argument should also be added to the API swagger docs in pkg/api/server/register_containers.go